DESCRIPTION

```diff
@@ -63,6 +63,6 @@ Imports:
     rlang,
     utils
 Depends: 
-    R (>= 3.5.0)
+    R (>= 3.6.0)
 Config/testthat/edition: 3
-Date: 2023-03-24
+Date: 2024-04-28
```

Comment:
Ah! Indeed. I already re-submitted, so let's see what CRAN says. The package passed the automated tests... So it might go through. I'll still update it in the meantime.